### PR TITLE
Display Page titles in the Next and Previous buttons (fixes #5101)

### LINKED ISSUE #5101: Display Page titles in the Next and Previous buttons
The documentation is built using Sphinx, and I think it'll be a good update if the page titles are written instead of `Prev` and `Next`.

### PR DESCRIPTION
**Pull request by obviyus**
## Description
Added the `footer.html` from [Sphinx-RTD theme](https://github.com/readthedocs/sphinx_rtd_theme/blob/master/sphinx_rtd_theme/footer.html) to override button text. 

## Issue / Bugzilla link
fix bug #5101

### CODE REVIEW

File: bedrock/firefox/templates/firefox/mobile/index.html [hunk 1]
```diff
@@ -68,12 +68,12 @@ <h3>{{ _('Get automatic privacy on mobile') }}</h3>
         <h2>{{ _('Block online trackers and invasive ads') }}</h2>
         <div class="c-private-cols">
           <div class="c-private-col">
-            <img src="{{ static('img/firefox/mobile/protocol/screen-private.svg') }}" height="331" width="163">
+            <img src="{{ static('img/firefox/mobile/protocol/screen-private.svg') }}" alt="" height="331" width="163">
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/mobile/index.html]: This doesn't seem related to this change? You may need to rebase your branch

File: docs/_templates/footer.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  
+    {%- if show_sphinx %}
+      {% set sphinx_web = '<a href="http://sphinx-doc.org/">Sphinx</a>' %}
+      {% set readthedocs_web = '<a href="https://readthedocs.org">Read the Docs</a>'  %}
+        {% trans sphinx_web=sphinx_web, readthedocs_web=readthedocs_web %}Built with {{ sphinx_web }} using a{% endtrans %} <a href="https://github.com/rtfd/sphinx_rtd_theme">{% trans %}theme{% endtrans %}</a> {% trans %}provided by {{ readthedocs_web }}{% endtrans %}.
+    {%- endif %}
+  
+    {%- block extrafooter %} {% endblock %}
+  
+  </footer>
```
Comment (alexgibson) [inline docs/_templates/footer.html]: Nit: new line at end of file please

**Comment by obviyus** [comment]
@alexgibson changes have been made
